### ISSUE TITLE
Color scans vs grayscale text scans

### ISSUE THREAD

**Issue by jonasrosland** [hitsave/scanning.guide#2]
From ubersaurus:
"it may be worthwhile to note that you don't need color or really high (600+) dpi if you're scanning a text document without images. Grayscale can work for those"

**Comment by KailoKyra** [comment]
I agree, For "Amstrad CPC Memoire ecrite", we scan books in 300dpi, 8 bit grayscale PNGs. 
It's more than enough to have a good image quality/resolution, and run it through an OCR.

**Comment by hardcorehubz** [comment]
Agreed, 300DPI should be sufficient for text only, could do 600DPI if it makes people feel better. :)
